internal/db/schema/migrations/postgres/2/84_oidc.up.sql

```diff
@@ -217,27 +202,26 @@ create or replace function
   returns trigger
 as $$
   begin
-    -- validate callback and signing alg
+    -- validate signing alg
     if old.state = 'inactive' and new.state != 'inactive' then
       perform 
       from 
         auth_oidc_method am
-       join auth_oidc_callback_url  cb    on am.public_id = cb.oidc_method_id 
        join auth_oidc_signing_alg   alg   on am.public_id = alg.oidc_method_id
```

Comment:
odd spacing now that there's only one join table there's no need to align them with spaces.